src/main/tachyon/tachyon-client.ts

```diff
@@ -16,6 +16,11 @@ export type TachyonClientRequestHandlers = {
     ) => Promise<Omit<GetCommands<"user", "server", "response", CommandId>, "type" | "commandId" | "messageId">>;
 };
 
+type TachyonError = {
+    error: string;
+    error_description: string;
+};
```

Comment:
This is not defined anywhere in the specification https://github.com/beyond-all-reason/tachyon/blob/master/docs/connection.md, I don't like depending on this not agreed on teiserver implementation detail. So, I would expect it to be first specified in protocol.

`TachyonError` is also too generic, it sounds like internal protocol error, when it's just initial websocket connection error.